tests/positive/files/units.go

```diff
@@ -114,3 +116,119 @@ func CreateEmptyDropinsUnit() types.Test {
 		ConfigMinVersion: configMinVersion,
 	}
 }
+
+func TestUnmaskUnit() types.Test {
+	name := "unit.unmask"
```

Comment:
Would be good to add a no-op unit as well (e.x. add another service with `mask: false` where we don't create the `/dev/null` symlink) to ensure that we're covering our bases.

Reply:
Added another test and also a description to the beginning of the test functions that were added.